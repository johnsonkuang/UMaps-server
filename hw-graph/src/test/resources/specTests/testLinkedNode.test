# Tests a node linked to itself
# Tests whether a node can be linked to itself via an edge

# Create graph
CreateGraph graph1

# Add node
AddNode graph1 n1

# Add edge connecting itself
AddEdge graph1 n1 n1 circularEdge

# List the nodes in graph1
ListNodes graph1

# List the children of n1
ListChildren graph1 n1