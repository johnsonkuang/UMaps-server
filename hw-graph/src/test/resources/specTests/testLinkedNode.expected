# Tests a node linked to itself
# Tests whether a node can be linked to itself via an edge

# Create graph
created graph graph1

# Add node
added node n1 to graph1

# Add edge connecting itself
added edge circularEdge from n1 to n1 in graph1

# List the nodes in graph1
graph1 contains: n1

# List the children of n1
the children of n1 in graph1 are: n1(circularEdge)